backend/app/app.auth.js

```diff
@@ -32,8 +32,53 @@ passport.deserializeUser((user, done) => {
 });
 
```

Comment:
What do you think about transform this object into a "module class" using static methods?

```javascript
class Auth {
  static initialize() { ... }

  static session() { ... }

  static isUserLoggedIn(req) { ... }

  static currentUser(req) { ... }

  static authenticate(opts) { ... }

  static login(req, user) { ... }

  static logout(req) { ... }
}

module.exports = Auth;
```

Look more organized in my opinion and don't change the usage (tests and imports).

Reply:
I don't like javascript classes for imports as much. My personal preference is to export only functions and explicit import what do you want.

What do you think to discuss about the backend structure / code on a issue?

Reply:
Sure man! =)

Reply:
I also prefer not to use classes in JavaScript, but either way doesn't make sense to me in having a class with only static methods, it run away from OOP.